tests/client_test.py

```diff
@@ -165,11 +165,10 @@ def test_client_array_dim_cmd_error(self):
         support multi-dimensional arrays of the given rank.
         """
         with pytest.raises(RuntimeError) as cm:
-            resp = generic_msg("reduce10D")
+            resp = generic_msg("sum<int64,10,1>")
 
         err_msg = (
-            f"Error: Command 'reduce10D' is not supported with the current server configuration as the maximum array dimensionality is {ak.client.get_max_array_rank()}. "
-            f"Please recompile with support for at least 10D arrays"
+            f"Unrecognized command: sum<int64,10,1>"
```

Comment:
This whole test can be removed now IMO (it's pretty specific to the registerND procedures).

Reply:
I removed it.  Thanks!